smart-frontend/app/src/components/uikit/Icon/Icon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import cn from 'classnames';
+import type { IconsNames } from './sprite';
+
+interface IconProps extends React.SVGAttributes<SVGSVGElement> {
+  size?: number;
+  name: IconsNames;
+}
+
+const Icon = React.forwardRef<SVGSVGElement, IconProps>(({ name, size = 12, className, ...props }, ref) => {
+  const classString = cn('icon', className);
+
+  return (
+    <>
```

Comment:
Please, remove Fragment

Reply:
Thanks! You are right. I tested different svg plugins and show some svg for one icon